modules/ocf_mysql/manifests/init.pp

```diff
@@ -1,30 +1,31 @@
 class ocf_mysql {
-  file {
-    # preseed root password
-    '/opt/share/puppet/mariadb-server-10.0.preseed':
-      mode      => '0600',
-      source    => 'puppet:///private/mariadb-server-10.0.preseed',
-      show_diff => false;
 
+  include ocf::ssl::default;
```

Comment:
maybe use `require` instead of `include` to ensure that the SSL files are in place before the service is configured/started (or add an explicit dependency later on)

otherwise i think the initial puppet run may fail?